problem with compiling the .bat files

Hi! I will really appreciate your help in solving the following problem:
After downloading the ONE 1.6.0 , when I tried compile.bat and one.bat commands from the command prompt got the following error messages-(jdk10.0.1 and jre 10.0.1 are successfully installed and tested in the computer)
1.
H:\the-one-1.6.0>compile.bat

H:\the-one-1.6.0>javac -extdirs lib/ core/*.java
javac: option -extdirs not allowed with target 1.10

H:\the-one-1.6.0>javac -extdirs lib/ movement/*.java
javac: option -extdirs not allowed with target 1.10
............
2.
H:\the-one-1.6.0>one.bat

H:\the-one-1.6.0>java -Xmx512M -cp .;lib/ECLA.jar;lib/DTNConsoleConnection.jar core.DTNSim
Error: Could not find or load main class core.DTNSim
Caused by: java.lang.ClassNotFoundException: core.DTNSim
Thanks in advance.
-Anika

--- a/compile.bat
+++ b/compile.bat
@@ -2,7 +2,7 @@ set targetdir=target
 
 IF NOT EXIST "%targetdir%" mkdir %targetdir%
 
-javac -sourcepath src -d %targetdir% -extdirs lib/ src/core/*.java src/movement/*.java src/report/*.java src/routing/*.java src/gui/*.java src/input/*.java src/applications/*.java src/interfaces/*.java
+javac -sourcepath src -d %targetdir% -cp lib/ECLA.jar:lib/DTNConsoleConnection.jar src/core/*.java src/movement/*.java src/report/*.java src/routing/*.java src/gui/*.java src/input/*.java src/applications/*.java src/interfaces/*.java
 
 
 

--- a/compile.sh
+++ b/compile.sh
@@ -2,7 +2,7 @@ targetdir=target
 
 if [ ! -d "$targetdir" ]; then mkdir $targetdir; fi
 
-javac -sourcepath src -d $targetdir -extdirs lib/ src/core/*.java src/movement/*.java src/report/*.java src/routing/*.java src/gui/*.java src/input/*.java src/applications/*.java src/interfaces/*.java
+javac -sourcepath src -d $targetdir -cp lib/ECLA.jar:lib/DTNConsoleConnection.jar src/core/*.java src/movement/*.java src/report/*.java src/routing/*.java src/gui/*.java src/input/*.java src/applications/*.java src/interfaces/*.java
 
 if [ ! -d "$targetdir/gui/buttonGraphics" ]; then cp -R src/gui/buttonGraphics target/gui/; fi
 	

